pkg/domain/nexpose/common.go

```diff
@@ -0,0 +1,17 @@
+package nexpose
```

Comment:
nit: please no files named `common`, `utils`, or `core`. These names provide no clear scope for what should be here and, therefore, become arbitrary data bags.

Instead, I'd recommend putting each one in a separate file named after the type within. Alternatively, we can put _all_ the nexpose types in one file.

Reply:
I agree. I copied it from another project, which passed review so I thought the team was okay with it. I'll change this and make a ticket to remove it from the other project.